ios/Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+ruby '~> 2.7.0'
```

Comment:
In the project README, we callout 2.7.4 as the version for ruby. Should we update that?

Reply:
fixed!